packages/babel-traverse/test/scope.js

```diff
@@ -73,6 +73,25 @@ describe("scope", () => {
       ).toBe("Identifier");
     });
 
+    describe("function paramater expression", function() {
+      it("should not has visibility of declarations inside function body", () => {
```

Comment:
Nit: after `should` you shouldn't use the third person but the base form of the verb (`should have`).

Reply:
Oops.